webrender/src/render_backend.rs

```diff
@@ -348,15 +348,28 @@ impl RenderBackend {
                             }
                         }
                         ApiMsg::ExternalEvent(evt) => {
-                            let mut notifier = self.notifier.lock();
+                            let notifier = self.notifier.lock();
                             notifier.unwrap()
                                     .as_mut()
                                     .unwrap()
                                     .external_event(evt);
                         }
+                        ApiMsg::ShutDown => {
+                            let notifier = self.notifier.lock();
```

Comment:
just wondering - why do we always have this `notifier` variable? can't we just go `self.notifier.lock().unwrap().as_mut()....` and so on?